### PR TITLE
USB upgrade UI improvements (release)

### PR DESCRIPTION
**Pull request by cbeighley**
Make instructions more clear and add a console print upon firmware version change.

`parse_version` was just returning None on firmware strings so I removed it. Now firmware version checking and firmware update nagging window works properly.

@denniszollo

### CODE REVIEW

File: piksi_tools/console/update_view.py [hunk 1]
```diff
@@ -646,6 +651,15 @@ def _compare_versions(self):
 
         fw_update_prompt.run()
 
+    # Check if firmware successfully upgraded and notify user if so.
+    if not self.fw_outdated and self.last_call_fw_version is not None and \
+        self.last_call_fw_version != local_stm_version:
+      print "Firmware successfully upgraded from %s to %s." %
```
Comment (denniszollo) [inline piksi_tools/console/update_view.py]: Can you make a write_call to the text window on the update tab for this message as well?

**Comment by cbeighley** [comment]
Holy crap how long does this take to build!?

**Comment by denniszollo** [comment]
it only takes 40 seconds if travis actually starts it

**Comment by denniszollo** [comment]
I think this is fine though because the previous build completed fine

**Comment by denniszollo** [comment]
LGTM merge at will

**Comment by cbeighley** [comment]
Hallelujah!

**Pull request merged**